src/AcceptanceTests/When_receiving_from_legacy_endpoint_using_conventions.cs

```diff
@@ -35,12 +36,14 @@ class SendingEndpoint : EndpointConfigurationBuilder
             public SendingEndpoint()
             {
                 EndpointSetup<DefaultServer>(c =>
-                    {
-                        // NServiceBus.Core encryption feature
-                        c.RijndaelEncryptionService(keyIdentifier, encryptionKey);
-                        c.Conventions().DefiningEncryptedPropertiesAs(p => p.Name == "Value");
-                    })
-                    .AddMapping<MessageWithEncryptedProperty>(typeof(ReceivingEndpoint));
+                {
+                    // NServiceBus.Core encryption feature
+                    c.RijndaelEncryptionService(keyIdentifier, encryptionKey);
+                    c.Conventions().DefiningEncryptedPropertiesAs(p => p.Name == "Value");
+                    c.UseTransport<MsmqTransport>()
```

Comment:
you can use this API to make this less dependent on the used transport: https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/Routing/When_replying_to_message.cs#L55